### PR TITLE
Listing details image size adjustment

### PR DESCRIPTION
**Pull request by henryhu1**
Make listing details image container uniform

### CODE REVIEW

File: android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt [hunk 1]
```diff
@@ -131,8 +133,8 @@ fun ListingDetailsPageView(
 							contentDescription = stringResource(id = R.string.listing_image),
 							contentScale = ContentScale.FillWidth,
```
Comment (henryhu1) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt]: `ContentScale` suggestions for the 250x250 container

* `ContentScale.Crop` center crop image and fit space
* `ContentScale.Fit` keep aspect ratio but have width or height whitespace for unfillable areas
* `ContentScale.FillBounds` fits space but lose aspect ratio

personally I like `ContentScale.Fit`

Comment (andr-dev) [reply android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt]: Sounds good, but let's make that 256 x 256. Always best to stick with powers / multiples of 2 unless you have a good reason to not.

Comment (henryhu1) [reply android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt]: another haiku 
multiples of two
sorry I was being faithful
to the figma mocks

edit - to be mathematically correct
yes powers of two
sorry I was being faithful
to the figma mocks

**Review by andr-dev — APPROVED** [review]
Nice!

**Review by abhaykshar — APPROVED** [review]
big!

File: android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt (hunk 1 above)
Comment (abhaykshar) [reply android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt]: Figma mocks are hot garbage idk who made those...

**Review by leeminkyu16 — APPROVED** [review]
Let's Gamble, Try Merging.

**Review by andr-dev — APPROVED** [review]
Love it :)

**Review by leeminkyu16 — APPROVED** [review]
Let's Gamble, Try Merging

**Pull request merged**